gtests/net/mptcp/unsupported/test_client_1.pkt

```diff
@@ -0,0 +1,30 @@
+// connect() function, connection initiated by the kernel
+--tolerance_usecs=100000
+`../../common/defaults.sh`
+
+
+//test 1 : Unsupported options
+0.0     socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0    setsockopt(3, SOL_TCP, TCP_INQ, [1], 4) = -1 EOPNOTSUPP (Operation not supported)
++0.0    getsockopt(3, SOL_TCP, TCP_INQ, [0], [4]) = -1 EOPNOTSUPP (Operation not supported)
+
+
+//testing mptcp for IP_TOS 
++0.0     socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 7
++0.0    setsockopt(7, SOL_IP, IP_TOS, [4], 4) = 0
++0.200  getsockopt(7, SOL_IP, IP_TOS, [4], [4]) = 0
+
++0.0    fcntl(7, F_GETFL) = 0x2 (flags O_RDWR)
++0.0    fcntl(7, F_SETFL, O_RDWR|O_NONBLOCK) = 0
+
+// Establish connection and check if TOS is set to default value
+
++0.0    connect(7, ..., ...) = -1 EINPROGRESS (Operation now in progress)
+
++0.0      >  [tos 0x04] S       0:0(0)                                 <mss 1460, sackOK, TS val 100 ecr 0,   nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
++0.0      <             S.      0:0(0)        ack 1     win 65535      <mss 1460, sackOK, TS val 700 ecr 100, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey=2]>
++0.0      >  [tos 0x04] .       1:1(0)        ack 1     win 256        <nop,nop,TS val 3081837704 ecr 700, mpcapable v1 flags[flag_h] key[ckey, skey]>
```

Comment:
can you align the TCP options with the previous ones to make it clearer to track the modifications reading them (like in the other files) please?

Also the ACK (`.`) is no aligned with the other ACKs above

Reply:
does this look right? 

```
+0.0  >  [tos 0x04]  S   0:0(0)                    <mss 1460, sackOK, TS val 100        ecr 0,   nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
+0.0  <              S.  0:0(0)  ack 1  win 65535  <mss 1460, sackOK, TS val 700        ecr 100, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey=2]>
+0.0  >  [tos 0x04]   .  1:1(0)  ack 1  win 256    <nop,      nop,    TS val 3081837704 ecr 700,                mpcapable v1 flags[flag_h] key[ckey, skey]>
``
```

Reply:
Oops, I forgot to reply: yes, perfect like that!

With a wide terminal (not limited to 80 chars), it is clearer to read and track the modifications between lines with this alignment in columns.